frontend/packages/topology/src/components/dropdowns/ApplicationSelector.tsx

```diff
@@ -70,7 +70,12 @@ const ApplicationSelector: React.FC<ApplicationSelectorProps> = ({
   ];
 
   const handleAppChange = (event) => {
-    setApplicationExists(availableApplications.current.includes(event.target.value));
+    setApplicationExists(availableApplications.current.includes(event.target.value.trim()));
+  };
+
+  const handleAppBlur = (event) => {
```

Comment:
Please add types to handleAppBlur and handleAppChange. :)

Reply:
Among all the other functions in this file, I am seeing only the parameters are given types.
Therefore, I have added type `(event: { target: { value: string; }; })` to the `event` parameter.